### PR TITLE
refactor: Remove includes; backport 16659

### PR DESCRIPTION
**Pull request by PastaPastaPasta**

### CODE REVIEW

File: src/qt/winshutdownmonitor.cpp [hunk 1]
```diff
@@ -1,10 +1,10 @@
 // Copyright (c) 2014 The Bitcoin Core developers
 // Distributed under the MIT software license, see the accompanying
 // file COPYING or http://www.opensource.org/licenses/mit-license.php.
-
 #include <qt/winshutdownmonitor.h>
 
 #if defined(Q_OS_WIN)
+
```
Comment (UdjinM6) [inline src/qt/winshutdownmonitor.cpp:7]: revert

**Review by UdjinM6 — CHANGES_REQUESTED** [review]
LGTM, one tiny suggestion + pls squash all the fixes into one "16659 followup" or smth like that

**Comment by PastaPastaPasta** [comment]
Done

**Review by UdjinM6 — APPROVED** [review]
utACK

**Pull request merged**